lxd/db/cluster/identities.go

```diff
@@ -63,8 +63,9 @@ import (
 type AuthMethod string
 
 const (
-	authMethodTLS  int64 = 1
-	authMethodOIDC int64 = 2
+	authMethodTLS    int64 = 1
+	authMethodOIDC   int64 = 2
+	authMethodBearer int64 = 3
```

Comment:
Should we leave out the type determination to the compiler rather than force it to be `int64`?  Maybe it's dependent of the underlying CPU arch but in the playground, it seems to pick `int` for such cases: https://go.dev/play/p/Uv8tSZ9taJ0

But maybe I'm splitting hairs here :)

Reply:
We tend to stick with int64 for DB values as a convention.